README.md

```diff
@@ -19,12 +19,16 @@ Once setup has been completed, you can start the server with
 script/server
 ```
 
-## Running the tests
+and the tests should pass
 
 ```bash
 script/test
 ```
 
+### Creating a development database
+
+See "PaaS to Local" at the end of the (Database Backup and Restore) guide.
```

Comment:
I think we can link this as [PaaS to Local](./doc/database-backup-and-restore.md#paas-to-local) `[PaaS to Local](./doc/database-backup-and-restore.md#paas-to-local)`. These links won't work from this comment because they're relative to the comment but they should work on commit.